File: hi_jim_android/shared/src/commonMain/kotlin/com/jim/hi_jim/shared/constants/UserConstants.kt
package com.jim.hi_jim.shared.constants

object UserConstants {
    // 하드코딩된 사용자 ID (본인과 여자친구)
    const val USER_1 = "user_jim"
    const val USER_2 = "user_girlfriend"

    // 현재 디바이스의 사용자 ID (각 워치마다 설정)
    // 실제로는 SharedPreferences나 다른 방식으로 저장하겠지만, 우선 하드코딩
    const val CURRENT_USER_ID = USER_2  // 이 값을 각 워치에서 다르게 설정
}

File: hi_jim_ios/hi_jim Watch App/UserConstants.swift
//
//  UserConstants.swift
//  hi_jim (Shared)
//
//  iOS와 Watch 앱 모두에서 사용하는 사용자 상수
//

import Foundation

// MARK: - User Constants
struct UserConstants {
    static let USER_1 = "user_jim"
    static let USER_2 = "user_girlfriend"

    // 각 기기에서 다르게 설정해야 함
    static var CURRENT_USER_ID: String {
        get {
            UserDefaults.standard.string(forKey: "currentUserId") ?? USER_2
        }
        set {
            UserDefaults.standard.set(newValue, forKey: "currentUserId")
        }
    }

    static var opponentUserId: String {
        return CURRENT_USER_ID == USER_1 ? USER_2 : USER_1
    }
}
